### PR TITLE
Automated dependency updates | Mon, 29 Oct 2018 12:02:50 +0000

### PR DESCRIPTION
**Pull request by akostibas**
Automated dependency update.

Two major things happened:
* `gobuffalo/uuid` was deprecated, so all instances of that import have been changed to `gofrs/uuid`
* The UUID packages (which are all just forks of the same original source,) have deprecated `uuid.Equal()` because the ["function provided no value"](https://github.com/gofrs/uuid/commit/0f0e88acbfb6e139991adfe98d8735b20d8b2c55). You can just do `uuidA == uuidB` and it works just fine. As such, I had to change all instances of `uuid.Equal()` accordingly.

### CODE REVIEW

**Comment by akostibas** [comment]
Builds failing with

```
#!/bin/bash -eo pipefail
make build
bin/check_go_version
go1.10 installed
touch .go_version.stamp
bin/check_gopath.sh
In order for go dependencies to be runnable, $GOPATH/bin must be in your $PATH
Please add /home/circleci/go/bin to your $PATH in your .bash_profile
Expected: /home/circleci/go/bin
Actual: /usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
dep ensure -vendor-only
# Unfortunately, dep ensure blows away ./vendor every time so these builds always take a while
go install ./vendor/github.com/golang/lint/golint # golint needs to be accessible for the pre-commit task to run, so `install` it
go build -i -o bin/gosec ./vendor/github.com/securego/gosec/cmd/gosec
go build -i -o bin/gin ./vendor/github.com/codegangsta/gin
go build -i -o bin/soda ./vendor/github.com/gobuffalo/pop/soda
go build -i -o bin/swagger ./vendor/github.com/go-swagger/go-swagger/cmd/swagger
touch .server_deps.stamp
go-bindata -o pkg/assets/assets.go -pkg assets pkg/paperwork/formtemplates/
bin/gen_server.sh
touch .server_generate.stamp
go build -i -o bin/webserver ./cmd/webserver
# github.com/transcom/mymove/pkg/models
pkg/models/upload.go:47:5: undefined: "github.com/transcom/mymove/vendor/github.com/gobuffalo/uuid".Equal
Makefile:92: recipe for target 'server_build' failed
make: *** [server_build] Error 2
Exited with code 2
```

**Comment by akostibas** [comment]
It looks like we need to be using a new package for UUIDs now. 🤡 

From GoBuffalo/UUID's page:
```
THIS PACKAGE IS DEPRECATED: please use github.com/gofrs/uuid instead.
```

File: pkg/uploader/uploader.go [hunk 1]
```diff
@@ -118,10 +118,8 @@ func (u *Uploader) DeleteUpload(upload *models.Upload) error {
 		return err
 	}
 
-	if err := models.DeleteUpload(u.db, upload); err != nil {
-		return err
-	}
-	return nil
+	err := models.DeleteUpload(u.db, upload)
```
Comment (ntwyman) [inline pkg/uploader/uploader.go]: What's wrong with 

```
return models.DeleteUpload(u.db, upload)
```

**Comment by dynamike** [comment]
There still appear to be references to github.com/gobuffalo/uuid

```
Gopkg.lock:  name = "github.com/gobuffalo/uuid"
Gopkg.lock:    "github.com/gobuffalo/uuid",
cmd/load_user_gen/main.go:	"github.com/gobuffalo/uuid"
cmd/generate_shipment_edi/main.go:	"github.com/gobuffalo/uuid"
cmd/make_office_user/main.go:	"github.com/gobuffalo/uuid"
cmd/generate_1203_form/main.go:	"github.com/gobuffalo/uuid"
cmd/make_tsp_user/main.go:	"github.com/gobuffalo/uuid"
cmd/paperwork/shipment_summary.go:	"github.com/gobuffalo/uuid"
internal/pkg/dutystationsloader/duty_stations_loader.go:	"github.com/gobuffalo/uuid"
internal/pkg/dutystationsloader/duty_stations_loader_test.go:	"github.com/gobuffalo/uuid"
```

**Pull request merged**